xamarin/Xamarin.Forms
# Fix SwipeItemView IsEnabled is not working

### Description of Change ###

The issue is not reproduced in 4.6. So this PR only includes a new SwipeView sample in Core Gallery to validate enable/disable SwipeItemView.

### Issues Resolved ### 

- fixes #10465

### API Changes ###

 None

### Platforms Affected ### 

- iOS
- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

![issue10465](https://user-images.githubusercontent.com/6755973/80383689-31d0c100-88a4-11ea-82d2-6948cf13ed12.gif)

### Testing Procedure ###
Launch Core Gallery and navigate to the SwipeView samples. Select _"SwipeItemView IsEnabled Gallery"_. Try enabling and disabling the SwipeItemView and verify that you cannot interact with the content if it is already disabled. 

### PR Checklist ###
